src/utils/xcode.js

```diff
@@ -86,6 +86,22 @@ function getXcodeVersion() {
   return 'unknown';
 }
 
+function removeUnusedXcodes() {
+  const recent = fs
+    .readdirSync(XcodeDir)
+    .map(xcode => {
+      const xcodePath = path.join(XcodeDir, xcode);
+      const { atime } = fs.statSync(xcodePath);
+      return { name: xcodePath, atime };
+    })
+    .sort((a, b) => b.atime - a.atime);
+
+  const { preserveXcode } = evmConfig.current();
+  for (const { name } of recent.slice(0, preserveXcode)) {
```

Comment:
I think this is backwards, the most recent are at the front and we want to preserve them.  So this should be:
```suggestion
  for (const { name } of recent.slice(preserveXcode)) {
```

Reply:
ope, you're right thank you 😶‍🌫️